flow/alerting/classifier.go

```diff
@@ -232,8 +232,14 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 			if strings.Contains(pgErr.Message, "invalid snapshot identifier") {
 				return ErrorNotifyInvalidSnapshotIdentifier, pgErrorInfo
 			}
-		case pgerrcode.TooManyConnections:
-			return ErrorNotifyConnectivity, pgErrorInfo // Maybe we can return something else?
+		case pgerrcode.TooManyConnections, // Maybe we can return something else?
+			pgerrcode.ConnectionException,
+			pgerrcode.ConnectionDoesNotExist,
+			pgerrcode.ConnectionFailure,
+			pgerrcode.SQLClientUnableToEstablishSQLConnection,
+			pgerrcode.SQLServerRejectedEstablishmentOfSQLConnection,
+			pgerrcode.ProtocolViolation:
```

Comment:
Not sure if `ProtocolViolation` is us

Reply:
I'm guessing it's a connection pooler complaining at us